### PR TITLE
feat: added activeDeadlineSeconds support for cron jobs

### PR DESCRIPTION
**Pull request by vpol**

### CODE REVIEW

**Comment by vpol** [comment]
@idoshamun I'm not 100% sure I've done it right, but that's what the [documentation](https://kubernetes.io/docs/concepts/workloads/controllers/job/) says.

File: src/suite/index.ts [hunk 1]
```diff
@@ -141,6 +141,7 @@ function deployCron(
     nameSuffix,
     schedule,
     concurrencyPolicy = 'Forbid',
+    activeDeadlineSeconds = 300,
```
Comment (idoshamun) [inline src/suite/index.ts:144]: Can you please remove the default? I'm afraid it might break existing cron jobs

Comment (vpol) [reply src/suite/index.ts:144]: do we have jobs running more than 5 minutes?

Comment (idoshamun) [reply src/suite/index.ts:144]: I dunno but I don't want to check it in production 😅

**Review by idoshamun — APPROVED** [review]

**Pull request merged**